templates/img/calendar.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16.284" height="14.929" viewBox="0 0 16.284 14.929"><defs><style>.a{fill:#f8651b;}.a,.b{fill-rule:evenodd;}.b{fill:#fff;}</style></defs><path class="a" d="M8.261,11.09H5.55v2.716H8.261Z" transform="translate(-2.834 -5.664)"/><path class="a" d="M16.581,19.41H13.87v2.716h2.711Z" transform="translate(-7.083 -9.913)"/><path class="b" d="M16.284,0V14.929H0V0ZM14.929,4.071H1.36v9.5H14.929Z"/><path class="a" d="M8.261,19.41H5.55v2.716H8.261Z" transform="translate(-2.834 -9.913)"/><path class="a" d="M24.9,11.09H22.19v2.716H24.9Z" transform="translate(-11.332 -5.664)"/><path class="a" d="M16.581,11.09H13.87v2.716h2.711Z" transform="translate(-7.083 -5.664)"/><path class="a" d="M24.9,22.126H22.19V19.41H24.9Z" transform="translate(-11.332 -9.913)"/></svg>
```

Comment:
Already in assets. If you need the same file on multiple locations could you use links?